ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -104,6 +105,9 @@ class NavigationControllerDemoController: DemoController {
             content.allowsCellSelection = true
         }
 
+        let searchBarView = accessoryView as! SearchBar
```

Comment:
This is crashing when you try to show without accessory:

Fatal error: Unexpectedly found nil while unwrapping an Optional value

Reply:
Fixed

---
In reply to: [470790975](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470790975) [](ancestors = 470790975)